README.md

```diff
@@ -25,6 +25,12 @@ First require the package:
 (require 'prettier-js)
 ```
 
+Or use [use-package](https://github.com/jwiegley/use-package)
+
```

Comment:
```suggestion
```

@jacksonrayhamilton is this what you mean?

Reply:
Yep.  I created https://github.com/prettier/prettier-emacs/pull/81 as an alternative to this PR

Reply:
Okay.